### ISSUE TITLE
Select previous version for new Angular Projects

### ISSUE THREAD

**Issue by KananCoder** [stackblitz/core#2826]
### **Is your feature request related to a problem? Please describe.**
When starting a new Angular project, it creates using the latest version of Angular.

### **Describe the solution you'd like**
I would like to be able to select a specific version; at minimum be able to pick (13.0, 14.0, 15.0, 16.0) and then upgrade to most recent minor version of the major version (i.e. 13.0 to 13.x).

### **Describe alternatives you've considered**
Downgrading the a Stackblitz development app is a pain.  Find older Angular examples and make them as a base line project, then fork from there.

### **Additional context**
None

**Comment by evheniyrz** [comment]
@KananCoder 
- you can use web container project. and through the console you can install the desired version of Angular. Having previously deleted the placeholdeo project. Of course with some dances with a tambourine, but very simple ones.
The console works as usual in the IDE.

**Comment by KananCoder** [comment]
@evheniyrz 
I have done that as well.  Thanks.  It just takes extra steps (entering npm commands) to start up the application as it doesn't auto start when I go the the StackBlitz project.

**Comment by mwaibel-go** [comment]
I agree downgrading Angular is a pain. Not only do you have to check which versions of TS and zone.js (and possibly other stuff) you need, you also need to manually edit the configuration files, so they work with a previous version.

**Comment by eFMSDev** [comment]
Please support create angular old version
